<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logs</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialong{
background-color:#f4f5fc

}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="updatebtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

border-radius: 25px;
color: white;
background-color: #24293e;
padding-top: 20px;
padding-bottom: 20px;






}</string>
       </property>
       <property name="text">
        <string>UPDATE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: white;
    font-size: 14px;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    padding: 5px;
    background: #24293e;
    border: 1px solid #dddddd;
    border-radius: 5px;
    text-align: center;
    qproperty-alignment: 'AlignCenter';
}

QLabel[warning=&quot;true&quot;] {
    color: #d9534f;
    background: #f2dede;
    border: 1px solid #ebccd1;
}

QLabel[success=&quot;true&quot;] {
    color: #3c763d;
    background: #dff0d8;
    border: 1px solid #d6e9c6;
}

QLabel[info=&quot;true&quot;] {
    color: #31708f;
    background: #d9edf7;
    border: 1px solid #bce8f1;
}
</string>
           </property>
           <property name="text">
            <string>SELECT SCHEDULE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCalendarWidget" name="calendarWidget">
           <property name="styleSheet">
            <string notr="true">/* General QCalendarWidget Style */
QCalendarWidget {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 8px;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 14px;
    color: #333333;
}

/* Navigation Bar */
QCalendarWidget QToolButton {
    color: #ffffff;
    background-color: #24293E;
    border: none;
    border-radius: 4px;
    padding: 5px;
}

QCalendarWidget QToolButton:hover {
    font-style: bold;
}

QCalendarWidget QToolButton::disabled {
    background-color: #9E9E9E;
}

/* Header (Month/Year) */
QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: #24293E;
    color: #ffffff;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
}

QCalendarWidget QToolButton::menu-indicator {
    image: none;
}

/* Day Headers (e.g., Mon, Tue) */
QCalendarWidget QTableView {
    border: none;
}

QCalendarWidget QHeaderView::section {
    color: #333333;
    background-color: #f2f2f2;
    border: none;
    padding: 5px;
}

/* Day Cells */
QCalendarWidget QTableView::item {
    border: none;
    padding: 5px;
}

QCalendarWidget QTableView::item:selected {
    background-color: #4CAF50;
    color: #ffffff;
}

QCalendarWidget QTableView::item:hover {
    background-color: #a5d6a7;
    color: #333333;
}

/* Today Highlight */
QCalendarWidget QWidget#qt_calendar_today {
    border: 1px solid #4CAF50;
    background-color: #ffffff;
    color: #4CAF50;
    border-radius: 4px;
}

QCalendarWidget QWidget#qt_calendar_today::hover {
    background-color: #e8f5e9;
    border: 1px solid #4CAF50;
}

/* Navigation Buttons */
QCalendarWidget QToolButton::left-arrow {
    image: url(:/icons/left-arrow.png); /* Ensure you have an appropriate icon */
}

QCalendarWidget QToolButton::right-arrow {
    image: url(:/icons/right-arrow.png); /* Ensure you have an appropriate icon */
}
QCalendarWidget{
	border: 1px solid black

}</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="okbtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

border-radius: 25px;
color: white;
background-color: #24293e;
padding-top: 20px;
padding-bottom: 20px;






}</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
